externs/browser/w3c_xml.js

```diff
@@ -400,6 +400,36 @@ XMLHttpRequest.prototype.onreadystatechange;
 XMLHttpRequest.prototype.onerror;
 
 /**
+ * @type {number}
+ * @see https://www.w3.org/TR/XMLHttpRequest/#states
+ */
+XMLHttpRequest.UNSENT = 0;
```

Comment:
No problem, done. But could you explain why some properties are assigned, like in this same file https://github.com/google/closure-compiler/blob/master/externs/browser/w3c_xml.js#L209 ? Is it bad behavior from before more strict writings of externs ?